build.sbt

```diff
@@ -133,3 +133,5 @@ lazy val uiTests = (project in file("ui-tests"))
   ) dependsOn backend
 
 RenameProject.settings
+
+fork in run := true
```

Comment:
Actually it's added itself when I've run the` activator` and after consulting [the docs](http://www.scala-sbt.org/0.13/docs/Running-Project-Code.html) I figured it couldn't hurt ;)
But after a closer look I see now it [can](https://stackoverflow.com/questions/31817729/activator-run-doesnt-work) [cause](https://github.com/typesafehub/activator/issues/1036) [problems](https://github.com/typesafehub/activator/issues/1060)